R/calcOutput.R

```diff
@@ -7,7 +7,7 @@
 #' @param type output type, e.g. "TauTotal". A list of all available source
 #' types can be retrieved with function \code{\link{getCalculations}}.
 #' @param aggregate Boolean indicating whether output data aggregation should be performed or not, "GLO" (or "glo") for aggregation to one global region,
-#' "REG+GLO" (or "regglo") for a combination of regional and global data.
+#' "REG+GLO" (or "regglo") for a combination of regional and global data, "REG+GLO+EXTRA" (or "reggloextra") same as "REG+GLO", but with an additional regional aggregation.
```

Comment:
It might be worthwhile to update the param description, but instead of adding another special case it would make more sense to explain the general possibility to have other aggregations and that you just have to mention the corresponding column name here. Please also do not introduce this alternative terminology which I had to introduce for backwards compatibility. It should be just the name of the column and "+" will allow to merge aggregation rules of more than one column

Reply:
Since you created the new convention and know it better, could you add the explanation please? Could you include a hint if the user has to provide ALL column names of the mapping, i.e. "RegionsCode" and "GLO" if the mapping contains these columns?

Reply:
How would you have to call calcCoutput if your mapping has the columns:  "X | CountryCode | RegionCode | meins"?

Reply:
you mean if you want to aggregate by all three? You would write aggregate = "country+region+meins"
Ideally, new mappings should have the columns X|country|region|meins, but your example would work as well due to the backwards compatibility fix in the code for CountryCode and RegionCode

Reply:
or if your data is on country level and you want to aggregate to region and meins it would be aggregate="region+meins"

Reply:
Thanks!